### PR TITLE
login page done

### PR DESCRIPTION
**Pull request by TJain02**
Screenshot (162)

### CODE REVIEW

**Review by DevNirah7 — APPROVED** [review]
Looks good to commit!

File: CSS-Components/login.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  #email{
+    width: 20rem;
+    padding: 0.3rem;
+    border-radius: 0.325rem;
+    border: 0.5px solid;
+  }
+
+  #password{
+    width: 20rem;
+    padding: 0.3rem;
```
Comment (Asypher00) [inline CSS-Components/login.css:84]: try using px for values below 10px

File: CSS-Components/login.css [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+  #remember-me{
+    cursor: pointer;
+  }
+
+  .forgot-pasword{
+    border: none;
+    background-color: transparent;
+    color: var(--indigo);
+    margin: 0.5rem;
```
Comment (Asypher00) [inline CSS-Components/login.css:115]: try using px for values below 10px

**Review by Asypher00 — COMMENTED** [review]
Suggested some changes. Otherwise the code looks good

**Pull request merged**